nautobot/extras/api/serializers.py

```diff
@@ -342,6 +342,12 @@ class ConfigContextSerializer(ValidatedModelSerializer, NotesSerializerMixin):
         many=True,
     )
     tags = serializers.SlugRelatedField(queryset=Tag.objects.all(), slug_field="slug", required=False, many=True)
+    dynamic_groups = SerializedPKRelatedField(
+        queryset=DynamicGroup.objects.all(),
+        serializer=NestedDynamicGroupSerializer(),
```

Comment:
This is problematic since you want to pass the class object, not an instance!

```suggestion
        serializer=NestedDynamicGroupSerializer,
```